src/validators/__tests__/is-string.test.ts

```diff
@@ -172,20 +179,20 @@ describe('IsString', () => {
     it('does not allow required string failing custom validation', () => {
         class Fixture {
             @IsString({
-                validate: (value: string): boolean => value === 'foo',
+                validate: ValidateIsFoo,
```

Comment:
From what I read in the source, the `class-validator` applies validations based on a lookup on a validator registry. Unfortunately, this seems to mean you can't create ad-hoc validators.